docs/usage.rst

```diff
@@ -138,6 +138,10 @@ value more suitable for you. This value, which defaults to `None`, will be passe
 `Django's bulk_create method <https://docs.djangoproject.com/en/stable/ref/models/querysets/#bulk-create>`_
 as the `batch_size` parameter.
 
+If you want to change how errors are handled you can create your own error handler
+and set ``MAILER_ERROR_HANDLER``. Your handler should return a connection (could be None)
+and an action (one of 'sent' or 'deferred').
```

Comment:
A good start but:
- please add that the value of setting `MAILER_ERROR_HANDLER` should be a string reference to a function that will be passed to Django's `import_string` helper and that `"mailer.engine.handle_backend_exception"` is the default value
- please add something like "a 2-tuple containing of" with regard to the return value
- please style `None` and `'sent'` and `'deferred'`  as code using double backticks
- please mention that `None` will cause a new connection to be created
- with regard to action, maybe add the word "taken" so that it's clear that it's about the past and what the handler did, not that it would influence the future in a way other than which counter to increment